### ISSUE TITLE
Getting exception on checkforupdate

### ISSUE THREAD

**Issue by LuizTokuhara** [megahertz/electron-simple-updater#60]
I have added electron-simple-updater in an angular6 + electron project.
The updater is running as I can console.log the options, but I got an error that I cant figure it out what it may be.

At the checkUpdateOnStart, autoDownload, channel and others I got this error:
```
arguments: [Exception: TypeError: 'caller', 'callee', and 'arguments' properties may not be accessed on strict mode functions or the arguments objects for calls to them at Function.remoteFunction (<anonymous>:2:14)]

caller:[Exception: TypeError: 'caller', 'callee', and 'arguments' properties may not be accessed on strict mode functions or the arguments objects for calls to them at Function.remoteFunction (<anonymous>:2:14)]
```

What I am using is this:
```
electronService.updater.init({
        url: 'URL',
        autoDownload: true,
        logger: true
});
console.log(electronService.updater.options);
```

Did anyone had this problem?

**Comment by LuizTokuhara** [comment]
Hi thanks!!!
Just one more question, I have uploaded the json and file but on start the application did not find the update.

My json:

> {
>   "linux-armv7l-prod": {
>     "update": "url/folder/file-1.0.1.AppImage",
>     "install": "url/folder/file-1.0.1.AppImage",
>     "version": "1.0.1"
>   }
> }

Is it correct?
I am testing in production build by the way.

Regards,
Luiz

**Comment by megahertz** [comment]
Yes, it seems correct. So the problem is in another place. Run your
application from console to view the logs.
